KlipTok.en-GB.json

```diff
@@ -1,116 +1,104 @@
 {
 	// Components
-
```

Comment:
This commit removed all empty lines.

I think it is better to keep oryginal en-US whitespace, this makes much easier to know if translation have all strings, by just looking at line numbers.

Reply:
I'm not overly concerned